xarray/tests/test_cftimeindex.py

```diff
@@ -1289,3 +1290,11 @@ def test_infer_freq(freq, calendar):
     indx = xr.cftime_range("2000-01-01", periods=3, freq=freq, calendar=calendar)
     out = xr.infer_freq(indx)
     assert out == freq
+
+
```

Comment:
Oh and you'll need this too so that the tests will still pass if the optional dependency of cftime is not installed.
```suggestion
@requires_cftime
```